agents/PCS-Large-Swap/src/utils.ts

```diff
@@ -0,0 +1,43 @@
+import { ethers, Finding, FindingType, FindingSeverity } from "forta-agent";
+import { BigNumber } from "ethers";
+import { getCreate2Address } from "@ethersproject/address";
+
+const getPancakePairCreate2Address = (
+  pancakeFactoryAddr: string,
+  token0: string,
+  token1: string,
+  initCode: string
+): string => {
+  const salt = ethers.utils.solidityKeccak256(["address", "address"], [token0, token1]);
+  return getCreate2Address(pancakeFactoryAddr, salt, initCode);
+};
+
+const createFinding = (
+  swapTokenIn: string,
+  swapTokenOut: string,
+  swapAmountIn: BigNumber,
+  swapAmountOut: BigNumber,
+  percentageTokenIn: BigNumber,
+  percentageTokenOut: BigNumber,
+  swap_recipient: string
```

Comment:
Can still see it in multiple places `swap_recipient`? 
Make it in camel case.

Reply:
DOne